common/lib/protocol-definitions/src/users.ts

```diff
@@ -8,4 +8,5 @@
  */
 export interface IUser {
     id: string;
+    name?: string;
```

Comment:
This protocol change seems like unrelated, or at least should be in separate PR?

Reply:
Yep! It would be a part of separate PR. I will remove this protocol change from here.